tests/unit/services/gmail/test_delete.py

```
@@ -0,0 +1,493 @@
+        mock_service = Mock()
+        mock_get_service.return_value = (mock_service, None)
+
+        # Mock list to return message IDs
+        mock_list = Mock()
+        mock_list.execute.return_value = {
+            "messages": [{"id": "msg1"}, {"id": "msg2"}, {"id": "msg3"}]
+        }
+        mock_service.users.return_value.messages.return_value.list.return_value = mock_list
+
+        # Mock batch request
+        def batch_execute_side_effect():
```

Ruff (0.14.10)

Missing return type annotation for private function `batch_execute_side_effect`

Add return type annotation: `None`

(ANN202)